docs/how_to/single_node.md

```diff
@@ -92,6 +92,11 @@ warn-level errors (prefixed with `[W]`) should not block bootstrapping.
 20:10:14.764771[I] successfully updated topology to 1 hosts
 ```
 
+The node also self-hosts its OpenAPI docs, outlining available endpoints. You can access this by
+going to `localhost:7201/api/v1/docs` in your browser.
+
+![OpenAPI Doc](redoc_screenshot.png)
```

Comment:
Nice! Dunno if this is feasible, but could we also host offline version of the API docs on m3db.github.io or w/e?

Reply:
Should be doable. I'll put up a separate diff for that.